src/Symfony/Bridge/Monolog/Tests/Handler/ElasticsearchLogstashHandlerTest.php

```diff
@@ -0,0 +1,119 @@
+            ++$callCount;
+
+            return new MockResponse();
+        };
+
+        $handler = new ElasticsearchLogstashHandlerWithHardCodedHostname('http://es:9200', 'log', new MockHttpClient($responseFactory));
+
+        $record = [
+            'message' => 'My info message',
+            'context' => [],
+            'level' => Logger::INFO,
+            'level_name' => Logger::getLevelName(Logger::INFO),
+            'channel' => 'app',
+            'datetime' => new \DateTime('2020-01-01T00:00:00+01:00'),
+            'extra' => [],
+        ];
+
+        $handler->handle($record);
+
+        $this->assertSame(1, $callCount);
+    }
+
+    public function testBandleBatch()
+    {
+        $callCount = 0;
+        $responseFactory = function ($method, $url, $options) use (&$callCount) {
+            $body = <<<EOBODY
+{"index":{"_index":"log","_type":"_doc"}}
+{"@timestamp":"2020-01-01T00:00:00.000000+01:00","@version":1,"host":"my hostname","message":"My info message","type":"application","channel":"app","level":"INFO"}
+
+{"index":{"_index":"log","_type":"_doc"}}
+{"@timestamp":"2020-01-01T00:00:01.000000+01:00","@version":1,"host":"my hostname","message":"My second message","type":"application","channel":"php","level":"WARNING"}
+
+
+EOBODY;
+
+            $this->assertSame('POST', $method);
+            $this->assertSame('http://es:9200/_bulk', $url);
+            $this->assertSame($body, $options['body']);
+            $this->assertSame('Content-Type: application/json', $options['normalized_headers']['content-type'][0]);
+            ++$callCount;
+
+            return new MockResponse();
+        };
+
+        $handler = new ElasticsearchLogstashHandlerWithHardCodedHostname('http://es:9200', 'log', new MockHttpClient($responseFactory));
+
+        $records = [
+            [
+                'message' => 'My info message',
+                'context' => [],
+                'level' => Logger::INFO,
+                'level_name' => Logger::getLevelName(Logger::INFO),
+                'channel' => 'app',
+                'datetime' => new \DateTime('2020-01-01T00:00:00+01:00'),
+                'extra' => [],
+            ],
+            [
+                'message' => 'My second message',
+                'context' => [],
+                'level' => Logger::WARNING,
+                'level_name' => Logger::getLevelName(Logger::WARNING),
+                'channel' => 'php',
+                'datetime' => new \DateTime('2020-01-01T00:00:01+01:00'),
+                'extra' => [],
+            ],
+        ];
+
+        $handler->handleBatch($records);
+
+        $this->assertSame(1, $callCount);
+    }
+}
+
+class ElasticsearchLogstashHandlerWithHardCodedHostname extends ElasticsearchLogstashHandler
+{
+    protected function getDefaultFormatter(): FormatterInterface
+    {
+        return new LogstashFormatter('application', 'my hostname', null, 'ctxt_', LogstashFormatter::V1);
```

Comment:
The DefaultFormatter that comes with the handler is not tested though.

Perhaps you want to consider adding a simple test with it.